src/plugin/heatmap/heatmaplayer.js

```diff
@@ -639,3 +639,12 @@ plugin.heatmap.Heatmap.drawShadow = function(context, sizeY, blur) {
   context.shadowColor = '#000';
   context.fill();
 };
+
+
+/**
+ * @enum {string}
+ * @const
+ */
+plugin.heatmap.Heatmap.LayerValues = {
+  CANVAS: 'canvas'
+};
```

Comment:
Rather than defining a whole new enum for this, you could tack it onto `plugin.heatmap.HeatmapField` instead. The properties on that are currently all keys for features, but a field for a layer doesn't hurt my head there.